### ISSUE TITLE
Support not rebasing the input binary view

### ISSUE THREAD

**Issue by clubby789** [Vector35/debugger#214]
For a PIE executable, executing under the debugger causes all addresses to permanently change to their rebased runtime values.

**Comment by xusheng6** [comment]
Yes the debugger rebases the binary. This may not always be the desired behavior. However, we do not have a better approach at the moment.

**Comment by xusheng6** [comment]
Related to https://github.com/Vector35/debugger/issues/477

**Comment by xusheng6** [comment]
Support for NOT rebasing the binary view is added. Right now the UI does not try to compensate for the fact that the binary view is not rebased, i.e., the functions will still be at its original address. The users can proceed as he wishes, or maybe rebase the binary view manually (https://github.com/Vector35/debugger/issues/821)
